qiskit/circuit/quantumcircuit.py

```diff
@@ -294,6 +294,8 @@ def from_instructions(
             | tuple[qiskit.circuit.Instruction, Iterable[Qubit], Iterable[Clbit]]
         ],
         *,
+        qubits: Optional[Iterable[Qubit]] = None,
+        clbits: Optional[Iterable[Qubit]] = None,
```

Comment:
```suggestion
        qubits: Iterable[Qubit] = (),
        clbits: Iterable[Clbit] = (),
```
typo in `clbits`, and since iterables have an immutable "empty" state, it's probably simpler just to use that rather than add a nullable check that could cause typing mistakes later.

Reply:
Done.